walletkit_client.go

```diff
@@ -346,3 +353,28 @@ func (m *walletKitClient) ListSweeps(ctx context.Context) ([]string, error) {
 	sweeps := resp.GetTransactionIds()
 	return sweeps.TransactionIds, nil
 }
+
+// BumpFee attempts to bump the fee of a transaction by spending one of its
+// outputs at the given fee rate. This essentially results in a
+// child-pays-for-parent (CPFP) scenario. If the given output has been used in a
+// previous BumpFee call, then a transaction replacing the previous is
+// broadcast, resulting in a replace-by-fee (RBF) scenario.
+func (m *walletKitClient) BumpFee(ctx context.Context, op wire.OutPoint,
+	feeRate chainfee.SatPerKWeight) error {
+
+	rpcCtx, cancel := context.WithTimeout(ctx, rpcTimeout)
+	defer cancel()
+
+	_, err := m.client.BumpFee(
+		m.walletKitMac.WithMacaroonAuth(rpcCtx),
+		&walletrpc.BumpFeeRequest{
+			Outpoint: &lnrpc.OutPoint{
+				TxidBytes:   op.Hash[:],
+				OutputIndex: op.Index,
+			},
+			SatPerByte: uint32(feeRate.FeePerKVByte() / 1000),
+			Force:      false,
```

Comment:
Should we allow this to be specified by the caller?

Reply:
We could, but I left it out as it wasn't necessary and seems mostly just used for anchors at the moment.